### PR TITLE
Crew incomplete

### LINKED ISSUE #1251: Allow suppression of missing cast/low crew boxes
We have `crew_incomplete: true` to enforce the boxes, but this doesn't currently extend to `crew_incomplete: false` for suppressing them. Use case: Festival of One Act Plays.

### PR DESCRIPTION
**Pull request by samozzy**
Closes #1251

### CODE REVIEW

File: _shows/15_16/steady_rain.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ date_start: 2016-05-30
 date_end: 2016-05-31
 venue: New Theatre Studio A
 
+crew_incomplete: false
```
Comment (samozzy) [inline _shows/15_16/steady_rain.md]: External shows have 'ignore_missing' set to true automagically, so don't need crew_incomplete setting. Thoughts?

File: _shows/15_16/the_importance_of_being_earnest_postgrad.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ date_start: 2016-05-25
 date_end: 2016-05-28
 venue: New Theatre
 
+crew_incomplete: false
```
Comment (samozzy) [inline _shows/15_16/the_importance_of_being_earnest_postgrad.md]: Postgrads have the same deal as external

File: _shows/17_18/dreaming_whilst_black.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 title: Dreaming Whilst Black
+playwright: Adjani Salmon and Ali Hughes
 season: StuFF
 season_sort: 450
 period: Spring
 venue: New Theatre
 date_start: 2018-06-16
 company: 4 Quarter Films
+
+crew_incomplete: false
```
Comment (samozzy) [inline _shows/17_18/dreaming_whilst_black.md]: Replace with `ignore_missing: true` on this one imo

File: _shows/15_16/steady_rain.md (hunk 1 above)
Comment (GingertronMk1) [reply _shows/15_16/steady_rain.md]: I was unaware of this, I feel like it's nice to have for consistency

**Pull request merged**